tests/test_opds_import.py

```diff
@@ -616,16 +645,16 @@ def test_resources_are_mirrored_on_import(self):
 
         http = DummyHTTPClient()
         http.queue_response(
-            200, content='I am 10557.epub.images',
-            media_type=Representation.EPUB_MEDIA_TYPE,
-        )
-        http.queue_response(
             200, content='I am 10441.epub.images',
             media_type=Representation.EPUB_MEDIA_TYPE
         )
         http.queue_response(
             200, content=svg, media_type=Representation.SVG_MEDIA_TYPE
         )
+        http.queue_response(
+            200, content='I am 10557.epub.images',
+            media_type=Representation.EPUB_MEDIA_TYPE,
+        )
```

Comment:
Actually, I have this wrong. Things get popped from the queue in reverse, so 10441 and 10557 are flipped here, but the test doesn't check the content, only the media type. What actually changed is that the epub for 10441 comes before the cover now. This is due to putting the circulation data inside the metadata, so the epub gets mirrored as part of metadata.apply instead of afterwards. metadata.apply does the circulation data first so when it processes links it can associate them with the pool.